src/module/TaskManagement/src/TaskManagement/Service/CreateTaskAssertion.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace TaskManagement\Service;
+
+
+use Zend\Permissions\Acl\Acl;
+use Zend\Permissions\Acl\Resource\ResourceInterface;
+use Zend\Permissions\Acl\Role\RoleInterface;
+use Zend\Permissions\Acl\Assertion\AssertionInterface;
+use Ora\User\User;
+
+class CreateTaskAssertion implements AssertionInterface
+{
+    private $loggedUser;
+    
+    //imposto il default a null su $loggedUser se la richiesta arriva senza che l'utente sia loggato
+    public function __construct(User $loggedUser = null) {
```

Comment:
ho provato a passare nel costruttore il servizio AuthorizationService. Ma la serializzazione fallisce ancora, con l'errore "Serialization of 'Closure' is not allowed" ... come se ci fosse una funzione anonima all'interno della classe...provo ad approfondire ulteriormente